resources/migrations/000_migrations.yaml

```diff
@@ -19,6 +7,14 @@ databaseChangeLog:
       name: timestamp_type
       value: timestamp(6)
       dbms: mysql,mariadb
+  - property:
+      name: blob.type
+      value: blob
+      dbms: mysql,h2,mariadb
+  - property:
+      name: blob.type
+      value: bytea
+      dbms: postgresql
```

Comment:
I moved these here so they could be with their friends, the other type properties.